packages/url-state-provider/src/index.ts

```diff
@@ -324,3 +324,5 @@ export {
   decode,
   encode,
 }
+
+export { encode as encodeV2, decode as decodeV2 } from "./v2"
```

Comment:
Question:
I understand why you did this in the context of a minor update. But why not make it a new major version instead and have the `encode` and `decode` exports be the V2 versions?

Reply:
The reason I did not created a major version just yet is because the new solution would run alongside the existing one so technically this is not a break change at the moment but after all apps are fully converted to the new URL structure then we can remove the old solution and release a major version as a breaking change and rename `encodeV2`, `decodeV2` exports. 
Moreover if we release a new major version we’ll have to make sure that any important change in the previous version(e.g vulnerabilities fix) will have to be separately released to the previous version as patch.